test/operand_capabilities_test.cpp

```diff
@@ -223,12 +223,12 @@ INSTANTIATE_TEST_SUITE_P(
     Dim, EnumCapabilityTest,
     Combine(Values(SPV_ENV_UNIVERSAL_1_0, SPV_ENV_UNIVERSAL_1_1),
             ValuesIn(std::vector<EnumCapabilityCase>{
-                CASE2(DIMENSIONALITY, Dim::Dim1D, Sampled1D, Image1D),
-                CASE3(DIMENSIONALITY, Dim::Dim2D, Kernel, Shader, ImageMSArray),
+                CASE1(DIMENSIONALITY, Dim::Dim1D, Sampled1D),
```

Comment:
These changes correspond to the change approved by the SPIR Working Group.
See https://github.com/KhronosGroup/SPIRV-Headers/pull/327